plans/cd4pe_job.pp

```diff
@@ -6,6 +6,7 @@
   Optional[Array[String[1]]]      $env_vars = undef,
   Optional[String[1]]             $docker_image = undef,
   Optional[Array[String[1]]]      $docker_run_args = undef,
+  Optional[String[1]]             $docker_pull_creds = undef,
```

Comment:
Did we want to bump CD4PE's bolt version to 3.12+, and mark this as sensitive? Or did you test that and it didn't work that well?

I don't think thats too necessary, as this plan is executed within the container, so it's never transmitted and stored in a PE database.

Also note, We don't need to release this change to the forge right away, as we consume this module from git in the CD4PE build process.

Reply:
I don't think I want to handle bumping CD4PE's bolt version currently, too many other things I need to spend time on.

Reply:
👍  it's getting bumped as part of a nother ticket in the sprint anyways, so can circle back later if we want to marking this as sensitive...but like i said since it's only used within the container only, should be fine

Reply:
I think when we raised that question previously, we also noted there were others that should be marked Sensitive.

Reply:
I might be mistaken though. Maybe it was just the CD4PE token.